src/components/Tooltip/TooltipRenderedOnPageBody.js

```diff
@@ -118,6 +118,7 @@ class TooltipRenderedOnPageBody extends React.PureComponent {
     }
 
     render() {
+        if (this.state.tooltipContentWidth > this.props.maxWidth) { this.updateTooltipContentWidth(); }
```

Comment:
Let's add a broken line so it's easy to read it. Also, please add comment why we are doing this and the problem we are trying to solve.

```suggestion
        if (this.state.tooltipContentWidth > this.props.maxWidth) {
            this.updateTooltipContentWidth();
        }
```